lib/install.js

```diff
@@ -21,6 +21,8 @@ var fs = require('graceful-fs')
   , minimatch = require('minimatch')
   , mkdir = require('mkdirp')
   , win = process.platform == 'win32'
+  , runtime = semver.parse(process.version).major < 1 ? 'node' : 'iojs'
```

Comment:
Longer-term, I'd really like to see iojs add an `"iojs"` property to `process.versions` so this test, which feels like a flaky heuristic that somebody, somewhere is going to forget about at some point, with unpredictable, but probably dire, consequences.